tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.md

```diff
@@ -100,12 +104,13 @@ cat: can't open 'mydata/test_nfs_file.txt': No such file or directory
 ```
 Error response from daemon: volume ${nfsNamedVolume} in use by
 ```
-* Steps 34 - 36 should result in success; step 36 should kill/drop the server
-* Step 37 should result in error with the following message:
+* Steps 34-38 should result in success
+* Steps 39 - 41 should result in success; step 41 should kill/drop the server
+* Step 42 should result in error with the following message:
 ```
 Server error from portlayer: unable to wait for process launch status:
```

Comment:
@mhagen-vmware The whole error in this case is:
`docker: Error response from daemon: Server error from portlayer: unable to wait for process launch status: Post https://10.192.228.204/sdk: context deadline exceeded.
`
And this happens because we kill the NFS server that housed the volume we were using.  @matthewavery Is there a way this type of error can be captured/tagged so we can have better error output?  Just want to know real quick before I create a ticket for this.